eo-maven-plugin/src/main/java/org/eolang/maven/OyRemote.java

```diff
@@ -24,16 +24,25 @@
 package org.eolang.maven;
 
 import com.jcabi.log.Logger;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
 import java.net.MalformedURLException;
 import java.net.URL;
+import java.nio.charset.StandardCharsets;
 import org.cactoos.Input;
 import org.cactoos.io.InputOf;
+import org.json.JSONException;
+import org.json.JSONObject;
 
 /**
  * The simple HTTP Objectionary server.
  *
  * @since 0.1
  */
+@SuppressWarnings({"PMD.AssignmentInOperand", "PMD.AvoidDuplicateLiterals"})
```

Comment:
@yasamprom `AssignmentInOperand` is a valid complaint, about code smell in the code; we don't suppress them